ocaml/parsing/ast_helper.ml

```diff
@@ -62,7 +62,7 @@ module Typ = struct
 
   let any ?loc ?attrs () = mk ?loc ?attrs Ptyp_any
   let var ?loc ?attrs a = mk ?loc ?attrs (Ptyp_var a)
-  let arrow ?loc ?attrs a b c = mk ?loc ?attrs (Ptyp_arrow (a, b, c))
+  let arrow ?loc ?attrs a b c d e = mk ?loc ?attrs (Ptyp_arrow (a, b, c, d, e))
```

Comment:
Maybe `d` and `e` can default to `[]`.
(I believe this function is not used anywhere in the tree, so it's a simple change)

Reply:
Hm. I'm not sure its worth breaking the convention that only `loc` and `attrs` are optional in these helper functions

Reply:
Fair - also it's not that important.